pkg/pipelinerunmetrics/metrics.go

```diff
@@ -163,6 +163,10 @@ func viewRegister(cfg *config.Metrics) error {
 	default:
 		return errors.New("invalid config for PipelinerunLevel: " + cfg.PipelinerunLevel)
 	}
+	runningPRTag, err := getRunningPipelineRunTags(cfg)
+	if err != nil {
+		return err
+	}
```

Comment:
Earlier logic is to get tags for PipelineRuns and new logic is to get RunningPipelineRun tags.

Now we have 2  options here.
extract older logic to new function just like we did it for running PRs or  keep the new logic in view register   only.
I will prefer former as that  offers more clean code.

Reply:
If the function is used in more than one place or if you write unit tests for it I see how it helps, otherwise I think we don't need a separate function